docs/source/tutorial/visualise_pipeline.md

```diff
@@ -248,7 +248,7 @@ The MatplotlibWriter dataset converts Matplotlib objects to image files. This me
 You can view Matplotlib charts in Kedro-Viz when you use the [Kedro MatplotLibWriter dataset](/kedro.extras.datasets.matplotlib.MatplotlibWriter). You must update the `src/requirements.txt` file in your Kedro project by adding the following dataset to enable Matplotlib for your project:
 
 ```
-kedro[matplotlib.MatplotlibWriter]==0.18.2
+kedro[matplotlib.MatplotlibWriter]==0.18.3
```

Comment:
```suggestion
kedro[matplotlib.MatplotlibWriter]~=0.18.3
```

Actually should it be `~=0.18.3`? @AntonyMilneQB

Reply:
I don't think it matters too much.